appveyor.yml

```diff
@@ -149,8 +149,8 @@ for:
   environment:
     ENVIRONMENTS_URL: https://downloads.mixxx.org/builds/buildserver/2.3.x-windows/
     ENVIRONMENTS_PATH: C:\mixxx-buildserver
-    ENVIRONMENT_NAME: 2.3-j00013-PLATFORM-CONFIGURATION-static-36f44bd2-minimal
-    QT_VERSION: 5.12.0
+    ENVIRONMENT_NAME: 2.3-j00019-PLATFORM-CONFIGURATION-static-55e94982-minimal
+    QT_VERSION: 5.14.2
```

Comment:
I was not aware these had to be updated. It would be nice if we could get this read from build/windows/golden_environment so we don't have to update it in two places.

Reply:
Possibly, it could be done. The section that failed looks like windows shell commands.
Also, it failed because the directory has been restored by the cache. Possibly the correct action is force a cleanup of that folder, but I'm not sure what's in there, so maybe simply not fail if it already exists.
Possibly for now, I would simply remove the && of this line
``` 
mkdir %ENVIRONMENTS_PATH% &&
```

Reply:
Yes replacing `&&` with `&` should work.

Reply:
> Possibly the correct action is force a cleanup of that folder

I don't think so. At any given time there are lots of open PRs. Every time the build environment is updated, they would keep downloading the old one until they merge master.

Reply:
Ping @rryan can you change
```
mkdir %ENVIRONMENTS_PATH% &&
```
to
```
mkdir %ENVIRONMENTS_PATH% &
```
in appveyor.yml so we can get this build going for testers.

Reply:
:+1: Build is going now: https://ci.appveyor.com/project/mixxxdj/mixxx-22dnk/builds/32151380/job/ntrf8gpgmm4j8l3s